mlrun/feature_store/feature_set.py

```diff
@@ -1030,6 +1030,56 @@ def deploy_ingestion_service(
             self, source, targets, name, run_config, verbose
         )
 
+    def check_relation(
+        self,
+        relation: Dict[str, Union[str, Entity]],
+        right_fset_fields,
+        ordered: bool = True,
+    ):
+        right_feature_set_entity_list = right_fset_fields.spec.entities
+
+        if (
+            all(ent in self.spec.entities for ent in right_feature_set_entity_list)
+            and len(right_feature_set_entity_list) == len(self.spec.entities)
+            and ordered
```

Comment:
how "ordered" is used? do we check the entities are in the same order?

Reply:
I removed it from this function